tests/utils/e2e/comparator/eltwise_kaldi.py

```diff
@@ -0,0 +1,78 @@
+import logging as log
+import sys
+
+import numpy as np
+
+from e2e_oss.common_utils.table_utils import make_table
+from .provider import ClassProvider
+
+
+class EltwiseKaldiComparator(ClassProvider):
```

Comment:
Kaldi, please remove it. I think eltwise comparation should be generic for all FWs